src/substrate/filters/labeler.ts

```diff
@@ -90,21 +90,21 @@ export const Label: LabelerFilter = (
       const { authorityId } = data;
       return {
         heading: 'Heartbeat Received',
-        label: ` A new heartbeat was received from ${fmtAddr(authorityId)}.`,
+        label: `A new heartbeat was received from ${fmtAddr(authorityId)}.`,
       };
     }
     case EventKind.SomeOffline: {
       const { sessionIndex } = data;
       return {
         heading: 'Some Offline',
-        label: `At the end of the session: ${sessionIndex}, at least one validator was found to be offline.`,
+        label: `At the end of session ${sessionIndex}, at least one validator was found to be offline.`,
```

Comment:
Maybe reword to "At least one validator was found to be offline at the end of session X". But we may want to avoid passive voice here and just say "At least one validator went offline during session X".